docs/content/guides/developer/stablecoins.mdx

```diff
@@ -37,195 +37,97 @@ USDY is a fully collateralized US dollar stablecoin issued by Ondo Finance, allo
+:::
 
 ### Prerequisites
 
-Before you start building the sample app to perform a USDC transfer, ensure you meet the following prerequisites:
+- Make sure you have some USDC tokens. Get Testnet tokens from Circle's [faucet](https://faucet.circle.com/).
 
-- [Node.js](https://nodejs.org/en) and [npm](https://www.npmjs.com/): Ensure that you have Node.js and npm installed on your machine. You can download and install Node.js from [nodejs.org](http://nodejs.org). npm comes with Node.js.
-- [Sui Wallet](https://chromewebstore.google.com/detail/sui-wallet/opcgpfmipidbgpenhmajoajpbobppdil): Install the Sui Wallet browser extension, set up your wallet, and connect it to the appropriate network. Ensure that your wallet is funded with:
-  - Some SUI tokens to cover transaction fees. (**Request Testnet SUI Tokens** from the **Wallet Settings** page)
-  - USDC tokens for the transfer. ([USDC Testnet Faucet](https://faucet.circle.com/))
+### USDC Stablecoin source code
 
-### Installation
-
-To begin, create a new project directory (`usdc-transfer-app`, for example) and initialize it with pnpm:
-
-```sh
-$ pnpm init
-```
+The USDC stablecoin source code is available in the [circlefin/stablecoin-sui](https://github.com/circlefin/stablecoin-sui/blob/master/packages/usdc/sources/usdc.move) repository.
 
-To keep a small footprint, the instruction uses [Parcel](https://parceljs.org/) to render the example React-based app. You can use a more robust framework, like Next.js, or a different package manager, but you must modify the steps appropriately. 
-
-```sh
-$ pnpm add react react-dom parcel
-```
-
-Install Sui dependencies next:
-
-```sh
-$ pnpm add @mysten/dapp-kit @mysten/sui @tanstack/react-query
-```
+### Import the USDC module in your Move package
 
-Finally, update your `package.json` to include the following script:
+To import the USDC module, add the following line to the `[dependencies]` section of your Move package's `Move.toml` file:
 
-```json
-"scripts": {
-    "start": "parcel index.html"
-  },
+```toml
+usdc = { git = "https://github.com/circlefin/stablecoin-sui.git", subdir = "packages/usdc", rev = "master" }
 ```
 
-### Create site structure
+After importing the module, your Move package should look like the following:
 
-At the root of your project directory, create an `index.html` file and save the following content to it:
+{@inject: examples/move/usdc_usage/Move.toml}
 
-<details>
-<summary>
-`index.html` file
-</summary>
-{@inject: examples/usdc-transfer-app/index.html}
-</details>
+:::warning
+The `usdc` package uses a specific version of the `sui` package. This will cause a version conflict with the `Sui` package in the Sui framework. You can override the version of the `Sui` package in your `Move.toml` file to use a different version. Add the `override = true` flag to the `Sui` package in your `Move.toml` file.
```

Comment:
```suggestion
The `usdc` package uses a specific version of the `sui` package, which causes a version conflict with the `Sui` package in the Sui framework. You can override the version of the `Sui` package in your `Move.toml` file to use a different version. Add the `override = true` flag to the `Sui` package in your `Move.toml` file.
```
fyi...and you might already know this but we change that local import in examples to testnet for sui dependency in the toml programmatically when we use that code import thingy.

Reply:
thanks for the heads up. This should be fine!